config/default/app_settings.json

```diff
@@ -1,454 +1,3205 @@
 {
-  "languages": [
```

Comment:
Why are there so many changes in this file?

Reply:
Accidentally committed settings created for testing. Reverted.